### DISCUSSION TITLE
Is there a class I can use to extract a list of authenticated org?

### DISCUSSION THREAD

**Discussion by jasmine-nguyen** [forcedotcom/cli#3053] [Q&A] [answered]
Similar to ```sf org list``` but in Typescript as I want to get this list and use it for a plugin i'm building? Thank you

**Comment by mdonnalley** [comment, marked as answer]
There's not a standalone class that provides that but you can look at our implementation here: https://github.com/salesforcecli/plugin-org/blob/main/src/commands/org/list.ts
